app/client/src/modules/scenes/private-room/private-room.component.ts

```diff
@@ -59,6 +60,54 @@ export const privateRoomComponent: ContainerComponent = () => {
   let $room;
   let $bubbleChat;
 
+  let $roomScene = container({
+    sortableChildren: true,
+    eventMode: EventMode.STATIC,
+  });
+
+  const margin = 100;
+  let isDragging = false;
+  let dragStart = { x: 0, y: 0 };
+  let containerStart = { x: 0, y: 0 };
+
+  global.events.on(TulipEvent.POINTER_DOWN, (event) => {
```

Comment:
We need to call this events on `mount` and remove it on `unmount` from `$container`. This is to prevent this events calling when the component is not loaded anymore. You can check this example https://github.com/openhotel/openhotel/blob/191b00bd3eb435d5e284646e66c0a78187efebcb/app/client/src/modules/interfaces/hot-bar.component.ts#L78-L87